ephemeral_storage_setup/utils.py

```diff
@@ -69,9 +70,18 @@ def mount(device_path, config):
         argv.append("-o")
         argv.append(",".join(config["mount_options"]))
 
-    argv.extend([device_path, config["mount_point"]["path"]])
+    mount_point_path = config["mount_point"]["path"]
+    argv.extend([device_path, mount_point_path])
     execute.simple(argv)
 
+    chown_config = config["mount_point"].get("chown")
+    if chown_config:
+        shutil.chown(
```

Comment:
Nice! I see now that os.chown doesn't support symbolic names. That means the "config" populate method doesn't work with them either. Something to fix later.